core-bundle/contao/templates/backend/be_two_factor.html5

```diff
@@ -39,6 +39,7 @@
         <input type="hidden" name="FORM_SUBMIT" value="tl_two_factor_disable">
         <input type="hidden" name="REQUEST_TOKEN" value="<?= $this->requestToken ?>">
         <div class="submit_container cf">
+          <a href="<?= $this->route('contao_backend', ['do' => 'security', 'act' => 'enable', 'ref' => $this->ref]) ?>" class="tl_submit" style="vertical-align: initial;"><?= $this->trans('MSC.edit') ?></a>
```

Comment:
not sure about `style="vertical-align: initial;"`, but there is a property `vertical-align: middle` for `a.tl_submit`.

Reply:
without: ![Bildschirmfoto 2025-02-04 um 22 36 56](https://github.com/user-attachments/assets/6923dccc-9d69-4c0a-87a5-05e9f28de0f3)